src/abstract/Multicall.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity 0.8.13;
+
+import { IMulticall } from "src/interfaces/IMulticall.sol";
+
+/// @notice Helper utility that enables calling multiple local methods in a single call.
+/// @author Modified from Uniswap (https://github.com/Uniswap/v3-periphery/blob/main/contracts/base/Multicall.sol)
+/// License-Identifier: GPL-2.0-or-later
+abstract contract Multicall is IMulticall {
+    function multicall(bytes[] calldata data) external payable returns (bytes[] memory results) {
```

Comment:
```suggestion
    function multicall(bytes[] calldata data) external payable returns (bytes[] memory rResults) {
```

Reply:
fixed all variables in this file and relevant interfaces